src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/SubmitDraftTest.java

```diff
@@ -86,22 +98,25 @@
 
     private SyaCaseWrapper draftAppeal;
 
+    @Autowired
+    private SubmitHelper submitHelper;
+
     @Before
     public void setUp() {
         baseURI = testUrl;
         useRelaxedHTTPSValidation();
         citizenToken = getIdamOauth2Token(username, password);
         citizenIdamTokens = IdamTokens.builder()
-                .idamOauth2Token(citizenToken)
-                .serviceAuthorization(authTokenGenerator.generate())
-                .userId(getUserId(citizenToken))
-                .build();
+            .idamOauth2Token(citizenToken)
+            .serviceAuthorization(authTokenGenerator.generate())
+            .userId(getUserId(citizenToken))
+            .build();
 
         userIdamTokens = idamService.getIdamTokens();
         draftAppeal = buildTestDraftAppeal();
     }
 
-    @BeforeEach
+    @After
     public void beforeEach() {
```

Comment:
Please rename this method to afterEach()

Reply:
done. Thanks